msteams-platform/concepts/device-capabilities/browser-device-permissions.md

```diff
@@ -9,27 +9,40 @@ ms.topic: how-to
 
-## Change in behavior
-If your application has declared that it needs device permissions in your [application manifest](native-device-permissions.md), then users will be shown an "app permissions" option where they can enable an app’s device permissions. The "app permissions" option can be found in personal apps, task module dialogs, and tabs in chats, channels or meetings.
+Teams apps that require device permissions, such as camera or microphone access, now require users to manually grant permission at a per app level in the web browser. 
+
+Previously, browser handled how to grant access permissions. Now these permissions are handled in Microsoft Teams. This has implications on how you design your application and if they require these permissions in the browser.
+
+## App permissions in the browser
+If your app needs device permissions as mentioned in your [application manifest](native-device-permissions.md#specify-permissions), **App permissions** option is enabled for app’s device permissions. The **App permissions** option is available in the following capabilities: 
+
+* [Personal apps and task module dialogs](#personal-apps-and-task-module-dialogs)
+* [Chats, channel, or meeting tabs](#chat-channel-or-meeting-tabs)
 
 ### Personal apps and task module dialogs
-The "app permissions" setting can be found in the top-right.
+The **App permissions** option is in the upper-right corner of the page.
+
+
 <img src="../../assets/images/tabs/apppermissions.png" alt="App permissions button" width="800"/>
 
-### Chat, channel or meeting tabs
-The "app permissions" setting can be found in the tab dropdown.
+### Chat, channel, or meeting tabs
+The **App permissions** option is in the dropdown of the tab.
+
+
 ![App permissions drop-down](../../assets/images/tabs/drop-downapppermissions.png)
 
-A user will need to enable these permissions in the browser for these permissions to take effect. Once a user changes an app’s device permissions in the browser, they will be prompted to reload the application in Teams. It is important that you make users aware of where to go in order to enable these permissions in Microsoft Teams.
+After the **App permissions** option is selected a popup displays, where the user can enable the permissions button.
+
+Users need to enable these permissions in the browser to access device capabilities for the app. After user changes the app’s device permissions in the browser, they're prompted to reload the application in Teams. It is important to ensure that the users are aware on how to enable these permissions in Microsoft Teams.
```

Comment:
Shouldn't the last sentence be added as a note?

"It is important to ensure that the users are aware on how to enable these permissions in Microsoft Teams."